src/styles/app-button/AppButton.tsx

```diff
@@ -0,0 +1,59 @@
+import { ForwardedRef, forwardRef } from "react";
+
+import classnames from "classnames";
+
+import "@/styles/app-button/AppButton.scss";
+import { AppButtonProps } from "@/styles/app-button/AppButton.types";
+import AppTypography from "@/styles/app-typography/AppTypography";
+import AppSpinner from "@/styles/app-spinner/AppSpinner";
+
+// TODO: add link variant
+const AppButton = forwardRef(function (
+  {
+    variant = "primary",
+    size = "md",
+    fullWidth = false,
+    isLoading = false,
+    disabled = false,
+    inline = false,
+    type = "button",
+    children,
+    prefixIcon: PrefixIcon,
+    postfixIcon: PostfixIcon,
```

Comment:
```suggestion
    prefixIcon,
    icon,
```

I would call props like follows